docs/src/administration/web/configure-environment.md

```diff
@@ -50,84 +50,88 @@ There are also additional options:
+![Environment email](/images/management-console/env-email.png "0.75")
 
-### Services
+### Hide from search engines
 
-To see more detailed information about a running service, click it in the tree or list.
+Under **Hide from search engines**, you can tell [search engines to ignore the site](../../environments/search-engine-visibility.md):
 
-The **Overview** gives you information about the given service.
-You can see the service version, the container size, and the disk size, if you've configured a persistent disk.
+![Environment search](/images/management-console/env-search.png "0.5")
 
-![Services: service overview](/images/management-console/service-tab/service-overview.png "0.5")
+### HTTP access control
 
-To see your `.platform/services.yaml` file that led to these routes, click **Configuration**.
+Under **HTTP access control**, you can [control access to your environment using HTTP methods](../../environments/http-access-control.md):
 
-## Environment settings
+![Settings control access with password and by IP](/images/management-console/settings-basics-access-control.png "0.5")
 
-To see settings for a given environment, click {{< icon settings >}} **Settings** within that environment.
+### Variables
 
-![Settings for an environment](/images/management-console/env-settings.png "0.75")
+Under **Variables**, you can define [environment variables](../../development/variables/_index.md):
 
-### Environment name
+![Configure Platform.sh environment variables](/images/management-console/settings-variables-environment.png "0.6")
 
-The first setting allows you to modify the name of the environment and view its parent environment.
+## Service information
 
-### Status
+For each environment, you can view information about how your routes, services, and apps are currently configured.
 
-The **Status** setting shows whether or not your environment is [active](../../other/glossary.md#active-environment).
-For non-Production environment, you can [change their status](../../environments/deactivate-environment.md).
+To do so, click **Services**.
+By default, you see configured routes.
 
-![Environment status](/images/management-console/env-status.png "0.5")
+### Routes
 
-### Outgoing emails
+The **Router** section shows a list of all the [routes configured on your environment](../../define-routes/_index.md).
+You can see each route's type and check if caching and server side includes have been enabled for it:
 
-You can allow your environment to [send outgoing emails](../../development/email.md).
+![Services: routes](/images/management-console/service-tab/routes.png "0.5")
 
-![Environment email](/images/management-console/env-email.png "0.75")
+To view the configuration file where your routes are set up, click **Configuration**.
 
-### Search engine visibility
+### Applications
 
-This setting enables you to tell [search engines to ignore the site](../../environments/search-engine-visibility.md).
+To see detailed information about an app container,
```

Comment:
:nail_care: Just so even the links are in the same place in each of these three sections. Not a necessary change, just a possibility for added consistency.
```suggestion
To see detailed information about an [app container](../../create-apps/),
```